pkg/local_object_storage/engine/put.go

```diff
@@ -126,10 +128,15 @@ func (e *StorageEngine) putToShard(sh *shard.Shard, ind int, pool util.WorkerPoo
 
 		_, err = sh.Put(putPrm)
 		if err != nil {
-			e.log.Warn("could not put object in shard",
-				zap.Stringer("shard", sh.ID()),
-				zap.String("error", err.Error()))
+			if errors.Is(err, shard.ErrReadOnlyMode) || errors.Is(err, blobstor.ErrNoPlaceFound) ||
```

Comment:
how about a helper func for that?

Reply:
It is custom in every case, I don't think it is worth it.